frontend/src/metabase/admin/settings/app/components/SettingsEditor/SettingsEditor.jsx

```diff
@@ -41,9 +43,9 @@ import { SettingsSection } from "./SettingsSection";
 
 const mapStateToProps = (state, props) => {
   return {
-    settings: getSettings(state, props),
-    settingValues: getSettingValues(state, props),
-    derivedSettingValues: getDerivedSettingValues(state, props),
+    settings: getAdminSettingsInfo(state, props),
+    settingValues: getAdminSettingValues(state, props),
+    derivedSettingValues: getSettings(state, props),
```

Comment:
🥴  this whole component is getting deleted soon, so a lot of these selectors can just get deleted